tools/README.md

```diff
@@ -0,0 +1,8 @@
+### Supported tools
+|                                                                      | web3js | Truffle | ethers | Hardhat |
+|----------------------------------------------------------------------|--------|---------|--------|---------|
+| Transfer HBARS                                                       |    ✅   |    ✅    |    ✅   |    ✅    |
+| Contract  Deployment                                                 |    ✅   |    ✅    |    ✅   |    ✅    |
+| Can use the contract instance after deploy without re-initialization |    ✅   |    ✅    |    ❌   |    ❌    |
+| Contract Method Call                                                 |    ✅   |    ✅    |    ✅   |    ✅    |
+| Contract Method Call Execution                                       |    ✅   |    ✅    |    ✅   |    ✅    |
```

Comment:
Can you add a note under the table highlighted the re-initialization issue.
You can also note the ethersjs bug so that there's transparency to the community.